geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/accumulo.scala

```diff
@@ -53,6 +53,9 @@ package object accumulo {
       val QUERY_EXACT_COUNT    = PropAndDefault("geomesa.force.count", "false")
       val QUERY_COST_TYPE      = PropAndDefault("geomesa.query.cost.type", null)
       val QUERY_TIMEOUT_MILLIS = PropAndDefault("geomesa.query.timeout.millis", null) // default is no timeout
+      // rough upper limit on the number of accumulo ranges we will generate per query
+      val SCAN_RANGES_TARGET   = PropAndDefault("geomesa.scan.ranges.target", "2000")
```

Comment:
i'm wondering if this should be exposed in the data store config as well or maybe we really need that geomesa-site.xml file :)